packages/flutter_tools/test/version_test.dart

```diff
@@ -201,13 +208,13 @@ void main() {
       Cache: () => mockCache,
     });
 
-    testUsingContext('ignores network issues', () async {
+    testUsingContext('does not print warning when unable to connect to server if not out of date', () async {
```

Comment:
There are now two possibilities when there are network issues - if the local repo is stale we'll still warn, but if it's not, we will not. So I duplicated this test to handle both; the first one has the same expectation (no warning) but uses a fresh local version, and the second one has an out of date repo and therefore expects the warning.